libHVCAENx527App/src/HVCAENx527MBBi.c

```diff
@@ -8,6 +8,9 @@
  * MultiBit Binary input record and output record device support routines.
  */
 
+#ifdef _WIN32
+#include <windows.h> /* we need to make sure EPICS callback.h is loaded after windows.h */
+#endif
```

Comment:
Document why we now need to include this extra header on the wiki - it's not obvious why this would be related to epics 7.

Reply:
EPICS defines CALLBACK as a structure, windows defines it as "__cdecl". If you need to include windows.h you must include it before epics redefines CALLBACK. You get a compile error like
```
  error C2059: syntax error: '__cdecl'
```
The caen includes windows.h and the previous version of EPICS must have somehow included windows.h in the group of headers that caen include, but now don't. If i include the epics osiSock.h it will pull in windows.h but i chose to explicitly include windows.h instead.   I'm guessing one of the standard epics headers used to automatically include osiSock.h but now doesn't

Reply:
Added to https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Compile-Errors